doc.js

```diff
@@ -225,41 +245,50 @@ define('doc', ['event'], function(event) {
 			},
 
 			'addClass' : function(clazz) {
+				var clazzList = clazz.split(' ');
 				this.each(function(el) {
-					if(!el.classList) {
-						if(!fallbackHasClass(el, clazz)) {
-							el.className += ' ' + clazz + ' ';
+					clazzList.forEach(function(clazz) {
+						if(!el.classList) {
```

Comment:
I believe that we can remove support for this, since all browsers that support the `forEach` method (used above) already support `classList`